knotx-stack-manager/src/main/packaging/bin/knotx.bat

```diff
@@ -0,0 +1,105 @@
+@if "%DEBUG%" == "" @echo off 
+@rem ##########################################################################
+@rem
+@rem  vertx startup script for Windows
+@rem
+@rem ##########################################################################
+
+@rem Set local scope for the variables with windows NT shell
+if "%OS%"=="Windows_NT" setlocal
+
+@rem Add default JVM options here. You can also use JAVA_OPTS and VERTX_OPTS to pass JVM options to this script.
+set DEFAULT_JVM_OPTS=-XX:+UseBiasedLocking -XX:BiasedLockingStartupDelay=0
+
+@rem enable remote debug port, uncomment the following
+@rem set JVM_DEBUG=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+

```

Comment:
Please add:

@rem You can specify the path to the vertx cache directory. If not specified a ${KNOTX_HOME}/.vertx is used
@rem VERTX_CACHE_DIR=

@rem You can specify path to your custom logger configuration file. If not specified a ${KNOTX_HOME}/conf/logback.xml is used
@rem KNOTX_LOGBACK_CONFIG =

@rem You can specify path to your custom Hazelcast cluser.xml file. If not specified a ${KNOTX_HOME}/conf/default-cluster.xml is used
@rem CLUSTER_CONFIG =

@rem You can specify hazelcast cluster options here. See http://docs.hazelcast.org/docs/3.6/manual/html-single/index.html#system-properties for the available options.
set HAZELCAST_OPTS=-Dhazelcast.max.no.heartbeat.seconds=5

@rem You can enable Vert.x metrics by uncommenting this line.
@rem METRICS_OPTS=-Dvertx.metrics.options.enabled=true -Dvertx.metrics.options.registryName=knotx-dropwizard-registry